dawn/src/dawn/Optimizer/Lowering.h

```diff
@@ -0,0 +1,44 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                          _
+//                         | |
+//                       __| | __ ___      ___ ___
+//                      / _` |/ _` \ \ /\ / / '_  |
+//                     | (_| | (_| |\ V  V /| | | |
+//                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#pragma once
+
+#include "dawn/Optimizer/Options.h"
+#include "dawn/Optimizer/PassManager.h"
+#include "dawn/Support/NonCopyable.h"
+#include <map>
+#include <memory>
+
+namespace dawn {
+
+struct SIR;
+namespace sir {
+struct Stencil;
+}
+namespace iir {
+class StencilInstantiation;
+}
+
+// TODO Remove this
```

Comment:
TODO up to date?

Reply:
I updated the documentation and removed the todo. Thanks for catching!